release/releaser.go

```diff
@@ -612,31 +612,31 @@ func (r *Releaser) releaseActionRegradeServices(services []flux.ServiceID, msg s
 				}
 			}
 
-			// Report individual service regrade results.
+			// Report individual service release results.
 			for _, service := range services {
 				namespace, serviceName := service.Components()
 				switch serviceName {
 				case FluxServiceName, FluxDaemonName:
 					continue
 				default:
 					if err := results[service]; err == nil { // no entry = nil error
-						rc.Instance.LogEvent(namespace, serviceName, "Regrade due to "+cause+": done")
+						rc.Instance.LogEvent(namespace, serviceName, msg+". done")
```

Comment:
Will these definitely start with `"Release"`, as the slack notifications expect?